clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -49,19 +52,28 @@
 (defresource create-client
```

Comment:
My SRP senses tingle a bit, looking at this resource, because its appears somewhat complex, but after looking at it for a while, I don't really have any concrete recommendations that I'm happy with. Everything in it largely deals with requesty and responsey responsibilities. Although that does imply two responsibilities. I dunno. I may just have to adjust my visual threshold when reading Clojure.